cluster/libvirt-coreos/user_data_minion.yml

```diff
@@ -17,7 +17,7 @@ coreos:
         --address=0.0.0.0 \
         --hostname-override=${NODE_IPS[$i]} \
         --cluster-domain=cluster.local \
-        --api-servers=http://${MASTER_IP}:8080 \
+        --kubeconfig=/opt/kubernetes/kubeconfig/kubelet.kubeconfig \
```

Comment:
is this path right? I don't see `/opt/kubernetes/kubeconfig` referenced anywhere else